infrastructure/environments/prod/variables.tf

```diff
@@ -33,4 +34,27 @@ variable "CLUSTER_ROUTE_NEXT_HOP" {
 variable "SUPPORT_EMAIL_ADDRESSES" {
   type        = list(string)
   description = "A list of email addresses for first line support alerts to be sent to."
+  default     = []
+}
+
+variable "KEYVAULT_ACCESS_CIDR_BLOCKS" {
+  type        = list(string)
+  description = "CIDR blocks representing whitelisted IPs to access keyvault"
+  default     = []
+}
+
+variable "KEYVAULT_NAME" {
+  description = "Name of keyvault where secrets stored"
+  default     = "mhra-prod"
+}
+
+variable "KEYVAULT_AUTHORISED_PERSON_IDS" {
+  type        = list(string)
+  description = "IDs of objects (people etc) to associate access policies for"
```

Comment:
as in the object id when you view their profile in azure ad?

Reply:
I'm not sure quite sure how you'd go about finding this ID for a given person in the Azure profile with our current level of access, but I imagine that's where you would find it. I'm not sure what the best way of sharing this list of ID's is without adding to source control, but you can find them by running `terraform plan` and seeing the object ID's that would be removed. Maybe it's fine for them to be public and I just add in to the variables file.